lua/net.lua

```diff
@@ -13,6 +17,20 @@ function M.setup_block()
 	conf.num_rx_queues = 3
 	conf.num_tx_queues = 3
 
+	conf.front.num_ports = #front_ports
+	conf.front.name = "front"
+	conf.front.pci_addrs = ffi.new("const char *[" .. #front_ports .. "]")
```

Comment:
This is what I did in the rebase that I pushed -- I was just commenting here that I couldn't come up with a better solution than having two representations of these strings in `struct net_config` -- one ephemerally for Lua to pass to C, and one permanently for C to use throughout the life of the program.